Compilation fails with Portenta H7 and Giga R1 - mbed core board ≥ 4.2.2

- Arduino IDE 2.3.6
- Arduino Mbed OS Giga Boards >= `4.2.2`

```
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_memp.o):(.rodata.memp_RAW_PCB+0x0): multiple definition of `memp_RAW_PCB'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/memp.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/../include/lwip/priv/memp_std.h:42: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_input':
lwip_raw.c:(.text.raw_input+0x0): multiple definition of `raw_input'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:136: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_bind':
lwip_raw.c:(.text.raw_bind+0x0): multiple definition of `raw_bind'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:224: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_bind_netif':
lwip_raw.c:(.text.raw_bind_netif+0x0): multiple definition of `raw_bind_netif'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:256: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_connect':
lwip_raw.c:(.text.raw_connect+0x0): multiple definition of `raw_connect'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:281: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_disconnect':
lwip_raw.c:(.text.raw_disconnect+0x0): multiple definition of `raw_disconnect'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:313: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_recv':
lwip_raw.c:(.text.raw_recv+0x0): multiple definition of `raw_recv'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:339: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_sendto_if_src':
lwip_raw.c:(.text.raw_sendto_if_src+0x0): multiple definition of `raw_sendto_if_src'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:422: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_sendto':
lwip_raw.c:(.text.raw_sendto+0x0): multiple definition of `raw_sendto'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:355: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_send':
lwip_raw.c:(.text.raw_send+0x0): multiple definition of `raw_send'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:548: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_remove':
lwip_raw.c:(.text.raw_remove+0x0): multiple definition of `raw_remove'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:566: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_new':
lwip_raw.c:(.text.raw_new+0x0): multiple definition of `raw_new'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:596: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_new_ip_type':
lwip_raw.c:(.text.raw_new_ip_type+0x0): multiple definition of `raw_new_ip_type'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:638: first defined here
/Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/variants/GIGA/libs/libmbed.a(lwip_raw.o): In function `raw_netif_ip_addr_changed':
lwip_raw.c:(.text.raw_netif_ip_addr_changed+0x0): multiple definition of `raw_netif_ip_addr_changed'
/Users/user/Library/Caches/arduino/sketches/774EA9D31C1621AED86FE09D30F5D5A1/libraries/Arduino_10BASE_T1S/lib/liblwip/core/raw.c.o:/Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S/src/lib/liblwip/core/raw.c:656: first defined here
collect2: error: ld returned 1 exit status
Using library Arduino_10BASE_T1S at version 0.1.0 in folder: /Users/user/Documents/Arduino/libraries/Arduino_10BASE_T1S 
Using library SPI in folder: /Users/user/Library/Arduino15/packages/arduino/hardware/mbed_giga/4.3.1/libraries/SPI (legacy)
exit status 1

Compilation error: exit status 1
```

--- a/src/lib/liblwip/cfg/lwipopts.h
+++ b/src/lib/liblwip/cfg/lwipopts.h
@@ -726,5 +726,19 @@ extern unsigned char debug_flags;
 #define etharp_output t1s_etharp_output
 #define icmp_input t1s_icmp_input
 #define icmp_dest_unreach t1s_icmp_dest_unreach
+#define memp_RAW_PCB t1s_memp_RAW_PCB
+#define raw_input t1s_raw_input
+#define raw_bind t1s_raw_bind
+#define raw_bind_netif t1s_raw_bind_netif
+#define raw_connect t1s_raw_connect
+#define raw_disconnect t1s_raw_disconnect
+#define raw_recv t1s_raw_recv
+#define raw_sendto_if_src t1s_raw_sendto_if_src
+#define raw_sendto t1s_raw_sendto
+#define raw_send t1s_raw_send
+#define raw_remove t1s_raw_remove
+#define raw_new t1s_raw_new
+#define raw_new_ip_type t1s_raw_new_ip_type
+#define raw_netif_ip_addr_changed t1s_raw_netif_ip_addr_changed
 
 #endif /* LWIP_LWIPOPTS_H */

